Src/FluentAssertions.Json/ObjectAssertionsExtensions.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using FluentAssertions.Equivalency;
+using FluentAssertions.Execution;
+using FluentAssertions.Primitives;
+using Newtonsoft.Json;
+
+// added to the root namespace to aid discovery
+// ReSharper disable CheckNamespace
+namespace FluentAssertions
```

Comment:
❌ Please use the same namespace as the rest of the code base.

Reply:
Fixed in https://github.com/fluentassertions/fluentassertions.json/pull/52/commits/c4ee603f6e7a2ec108af29611af0fd1f715c6715